defaults/k8s_eks_config.yaml

```diff
@@ -43,7 +43,7 @@ n_monitor_nodes: 1
 n_loaders: 1
 n_db_nodes: 4
 k8s_n_scylla_pods_per_cluster: 3
-k8s_n_auxiliary_nodes: 2
+k8s_n_auxiliary_nodes: 3
```

Comment:
this mean all the jobs not using sni_proxy would have one extra unused node.

I would argue for building a server configuration, that would hold those parameter need for a server job.

Reply:
> this mean all the jobs not using sni_proxy would have one extra unused node.

False. It will be used, it will not be loaded that much as it is now.

> I would argue for building a server configuration, that would hold those parameter need for a server job.

I didn't understand this statement at all.

Reply:
> > this mean all the jobs not using sni_proxy would have one extra unused node.
> 
> False. It will be used, it will not be loaded that much as it is now. It directly influences workability of all the pods on the default node pool. Faster work of mgmt and other side-effects...

if it wasn't for serverless, we wouldn't have enlarged this number, right ? 
If it was o.k. for us running the cases we had with pool of 2 nodes, we can keep working with that

> 
> > I would argue for building a server configuration, that would hold those parameter need for a server job.
> 
> I didn't understand this statement at all.

We should make a configuration for serverless, with the relevant numbers.

Reply:
It is strange to me to read it when we have this:

![Screenshot from 2023-01-23 09-57-31](https://user-images.githubusercontent.com/3796058/213990489-9553cf85-9b28-4fab-8117-e3e4b5c272a7.png)

Reply:
when at the beginning it was one single value, it made sense to no complicate it with multiple configuration files
now that we see more and more values we would need to set for the sake of serverless/ingress configuration.

revisiting this idea might be in order.

altering all the defaults to match what we need for serverless, for functional test I wouldn't mind much.
but now it's effects all of the EKS based test, and it's a bit more problematic, and cost for longevity, we should think how to keep that at bay (k8s setup is moe expensive anyhow, for various reasons, and serverless gonna be more cause of loadblancers)